### DISCUSSION TITLE
strelka

### DISCUSSION THREAD

**Discussion by ghostrider9899** [Security-Onion-Solutions/securityonion#5215] [Unsupported Versions] [answered]
Hi, 
I have a question regarding Strelka logs. I am reading through the format and was wondering, if the file scan detected a something bad from a Yara signature, Does it show it in the logs? I new to Strelka and was wondering if anybody has an example of a malicious alert from the Strelka logs?

**Comment by ghostrider9899** [comment]
High i was wondering if i can get a help on this quickly

**Comment by dougburks** [comment, marked as answer]
I've added some screenshots to the Strelka documentation:
https://docs.securityonion.net/en/2.3/strelka.html
